decidim-meetings/app/commands/decidim/meetings/withdraw_meeting.rb

```diff
@@ -32,7 +33,7 @@ def call
       private
 
       def change_meeting_state_to_withdrawn
-        @meeting.update state: "withdrawn"
+        @meeting.update(withdrawn_at: Time.zone.now)
```

Comment:
let's have a consistent (single) way of withdrawing a meeting. 
```suggestion
        @meeting.withdraw!
```